thirdparty/custom-overlay/whisper/portfile.cmake

```diff
@@ -0,0 +1,29 @@
+vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO Apra-Labs/whisper.cpp
+    REF 74ad92406e58eee4c0618d8749df48aae91194cc #v1.5.2
+    SHA512 897479792902f4cb2f36ef199c699b1cc1472c3b88898027230528b8ad1f27a0de1548fdf8eca44981e558c84cd37c8091375647d208195b1b7443a18a6cfa7e  # This is a temporary value. We will modify this value in the next section.
+    HEAD_REF kj/add-Config-for-vcpkg
```

Comment:
curious what does this line mean ? is this a branch or label ?

Reply:
This is a branch. Once all development is complete I will move this to HEAD_REF ApraPipesBuild